app-frontend/src/app/pages/projects/edit/annotate/import/import.scss

```diff
@@ -0,0 +1,21 @@
+@import "../../../../../../assets/styles/sass/settings/_colors.scss";
+
+#btn-upload {
+    width: 0.1px;
+	height: 0.1px;
+	opacity: 0;
+	overflow: hidden;
+	position: absolute;
+	z-index: -1;
+}
+
+#btn-upload-label {
+  color: $white;
+  margin-bottom: 0px;
+  cursor: pointer;
+  font-weight: 500;
+}
+
+div.dropdown.btn-group.fixedwidth.upload {
```

Comment:
Pretty serious selector here. You could probably come up with a better class name like `upload-btn-group`. Not critical by any means.